### PR TITLE
Rename lib/babel.js to lib/test-worker.js

### PR DESCRIPTION
**Pull request by ariporad**
See #374.

### CODE REVIEW

File: test/fixture/source-map-initial.js [hunk 1]
```diff
@@ -1,19 +1,9 @@
...
+import { mapFile } from 'source-map-fixtures';
+import test from '../../';
+const fixture = mapFile('throws').require();
+test('throw an uncaught exception', t => {
   setImmediate(run);
 });
-var run = function run() {
-  return fixture.run();
-};
+const run = () => fixture.run();
```
Comment (sindresorhus) [inline test/fixture/source-map-initial.js]: Why change this? Looks unrelated to the intent of this pull request.

Comment (ariporad) [inline test/fixture/source-map-initial.js]: I didn't... I have absolutely no clue why that's there.

Comment (ariporad) [inline test/fixture/source-map-initial.js]: I didn't... I have absolutely no clue why that's there.

Comment (sindresorhus) [inline test/fixture/source-map-initial.js]: Can you remove that change? And the below file change too.

**Comment by ariporad** [comment]
Ok @sindresorhus, that should be fixed now.

I have no clue why that happened... It looks like WebStorm automagically un-babel'd it.

**Comment by jamestalmage** [comment]
WebStorm asks you about adding a file watcher whenever it sees ES6 code. If you accidentally click yes, this can happen. I keep meaning to figure out where the setting is just to turn that off.

**Comment by ariporad** [comment]
No, I get that. But it appears that WebStorm _un-_ babel'd the code. It's
very strange.

**Comment by vdemedes** [comment]
When I see `child.js` I am asking myself a question "child of what?". In my opinion, it is just as unclear as `babel.js`. How about `forked-process.js`, `worker.js`, `test-worker.js`?

**Comment by ariporad** [comment]
@vdemedes: what about "subprocess.js"?

**Comment by vdemedes** [comment]
@ariporad `subprocess.js` works too. Let's see what other guys think.

**Comment by jamestalmage** [comment]
`test-file-launcher.js`

**Comment by jamestalmage** [comment]
Also, please paint the bikeshed green.

**Comment by vdemedes** [comment]
@jamestalmage a bit long, I think.

**Comment by jamestalmage** [comment]
`test-launcher.js` or `test-worker.js`

**Comment by ariporad** [comment]
@jamestalmage, @vdemedes: I still like subprocess.js or child.js.


And this bikeshed is to be blue.

**Comment by vdemedes** [comment]
@jamestalmage++

**Comment by sindresorhus** [comment]
:+1: `test-worker.js`

**Comment by ariporad** [comment]
Ok, I'll change it to `test-worker` and update the commit.
